frontend/src/axios.js

```diff
@@ -1,13 +1,14 @@
 import axios from "axios";
 
 export default function api(config) {
-  config = config === undefined && {};
+  config = config === undefined ? {} : config;
   let jwtToken;
   if (config.sendToken) {
     jwtToken = localStorage.getItem("token");
   }
-  return axios.create({
+  const pkg = {
```

Comment:
this is a more readable way nice remake but what does "pkg" mean?

Reply:
package, I couldn't come up with a better name and `package` is apparently a js keyword. I'm open to other ideas.

Reply:
might be defaultConfig or axiosConfig

Reply:
I'll go with defaultConfig.